recipes/python-ldap/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "python-ldap" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 41975e79406502c092732c57ef0c2c2eb318d91e8e765f81f5d4ab6c1db727c5
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - pyasn1
+    - pyasn1-modules
+    - openldap
+  run:
+    - python
+    - pyasn1
+    - pyasn1-modules
+    - openldap
+
+test:
+  imports:
+    - ldap
+    - ldap.controls
+    - ldap.extop
+    - ldap.schema
+    # - slapdtest
```

Comment:
Do you need this comment or shall we remove it?

Reply:
It we decide to add slapd to openldap it may be handy to uncomment